ynput/ayon-core
# Blender: Deadline asset dependencies

## Changelog Description
Implemented deadline file dependencies for Blender. Images and alembic files are set as dependencies.

## Testing notes:
1. Create a render in Blender for a scene that had alembic files (with a Transform Cache modifier) and/or textures.
2. Publish the render to deadline.
3. Check if the job in deadline has the files as dependencies:
![image](https://github.com/ynput/ayon-core/assets/1087869/afafe1b1-3d3b-4888-a844-43c469a1b2f9)
